### ISSUE TITLE
Why don't you use cowboy?

### ISSUE THREAD

**Issue by si14** [flashingpumpkin/spooky#4]
https://github.com/extend/cowboy it's way better written and documented.

**Comment by flashingpumpkin** [comment]
Before starting off with Spooky I was reading the code of Nitrogen and Chicago Boss, neither of which have had support for Cowboy at that time, if I remember correctly.

It would make sense to abstract the actual HTTP lib away though so one can freely chose.

**Comment by si14** [comment]
Yeah, cowboy is kinda new, but it's already stable. I recommend you to suggest using it.
